server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import { CrmService } from '../../crm/crm.service';
+import { PasFormService } from '../pas-form/pas-form.service';
+import { PACKAGE_ATTRS } from '../packages.controller';
+
+@Injectable()
+export class RwcdsFormService {
+  constructor(
+    private readonly crmService: CrmService,
+    private readonly pasFormService: PasFormService,
+  ) {}
+
+  async find(id) {
```

Comment:
I think this function does too much to simply be called a generic `find` function.  Should be something like `getPrefilled`

Reply:
Hmmm `getPrefilled` seems wrong to me — don't we always want it to do that, no matter what? AFAICT, it's a full part of the RWCDSForm "model" — there isn't really an exception where we _don't_ want it prefilled with some data.

If we had proper Model objects, it could be handled there. I know this is possible in Rails, but here we are.

Reply:
That's an interesting point, that we can lean on the assumption that the consumers of the API will never want an RWCDS without pre-filled fields. 

I guess that's like the backend to pre-filling the PAS name from the Project name